nuxeo-retention-core/src/main/resources/schema/retention_rule.xsd

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<xs:schema targetNamespace="http://www.nuxeo.org/ecm/schemas/retention_rule/"
+  xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:nxs="http://www.nuxeo.org/ecm/schemas/retention_rule/">
+
+  <xs:complexType name="retention_rule_condition">
+    <xs:sequence>
+      <xs:element name="expression" type="xs:string" />
+      <xs:element name="event" type="xs:string" />
+    </xs:sequence>
+  </xs:complexType>
+
+  <xs:element name="ruleId" type="xs:string" />
+  <xs:element name="beginCondition" type="nxs:retention_rule_condition" />
+  <xs:element name="endCondition" type="nxs:retention_rule_condition" />
+  <xs:element name="beginDelayInMillis" type="xs:long" />
```

Comment:
this will be an internal method -> the user creates a retention rule specifying a human readable format and than we translate to this ( studio for XML rules, UI for dynamic rules), I don't see the point of parsing /etc for something  internal that can be transformed